app/app_test.go

```diff
@@ -81,8 +82,9 @@ func (at *appTest) reset() {
 	require.True(at.t, resabci.IsOK(), resabci)
 }
 
-func getBalance(key basecoin.Actor, state state.KVStore) (coin.Coins, error) {
-	acct, err := coin.NewAccountant("").GetAccount(state, key)
+func getBalance(key basecoin.Actor, store state.KVStore) (coin.Coins, error) {
+	cspace := stack.PrefixedStore(coin.NameCoin, store)
+	acct, err := coin.GetAccount(cspace, key)
```

Comment:
This only works here, as the test code has access to the global store.  It has to restrict itself to a subset so it can call the module normally. 

When the module gets a handle inside DeliverTx,CheckTx,or SetOption, it is already the limited view.